framework/src/outputs/VTKOutput.C

```diff
@@ -35,6 +33,9 @@ validParams<VTKOutput>()
 VTKOutput::VTKOutput(const InputParameters & parameters)
   : OversampleOutput(parameters), _binary(getParam<bool>("binary"))
 {
+#ifndef LIBMESH_HAVE_VTK
+  mooseError("libMesh not configured with VTK");
```

Comment:
"You are trying to use VTK output, but your libMesh was not configured with VTK support. If you want to use it, please reconfigure libMesh to support it."

Reply:
Unfortunately, we cannot give more specific details on how to configure libMesh, because sometimes they may need to load a module, or set they environment manually or pass `--with-vtk-input`, `--with-vtk-lib`.